src/browser/components/buttons/ConfirmationButton.jsx

```diff
@@ -51,7 +51,7 @@ export class ConfirmationButton extends Component {
   }
 
   render () {
-    if (this.state.requested) {
+    if (this.state.requested || this.props.requested) {
```

Comment:
revert the change and if required come up with your own component which has confirm and cancel button.

Reply:
Added a new component